tests/console/gpg.pm

```diff
@@ -172,7 +177,7 @@ sub run {
         libgcrypt20 => '1.9.0',
         'libgcrypt20-hmac' => '1.9.0'
     };
-    zypper_call("in " . join(' ', keys %$pkg_list));
+    zypper_call("in --allow-downgrade " . join(' ', keys %$pkg_list));
```

Comment:
this problem is very specific to situation which we have ONLY when we testing "Latest" when image is newer than state of the repo , in any maintenance update I would treat downgrade as a bug or at least some which should be reviewed by human , so please add check for BETA=1